proposals/extensions.md

```diff
@@ -377,7 +369,7 @@ static class IEnumerableExtensions
         public async Task<int> SumAsync() { ... }
     }
 
-    public void Method() { ... }
+    public void Method2() { ... }
```

Comment:
It looks like there is no need to rename this method because it doesn't cause a conflict in the given scenario even when speakable names are used. Also, the method should probably be declared `static` since it is in a static class.

Reply:
The purpose of rename is to make this more readable (avoid any confusion between the two methods with the same name).